### ISSUE TITLE
Adding additional pages to the website

### ISSUE THREAD

**Issue by CathyHajo** [JonathanReeve/sanger#81]
Esther would like to add some descriptive pages to the site, but we are not sure how to add them to the Digital Ocean site, and how to link them to the navigation on the digital edition. Currently the links go to the Sanger Project website, rather than a dedicated digital edition page, and we would like to change that.

**Comment by EstherKatz** [comment]
Yes, i would like to add additional pages!

**Comment by EstherKatz** [comment]
Yes, I want to add additional pages to the site. How do I do it?

**Comment by JonathanReeve** [comment]
If you want, you can write the new pages in markdown or plain text, and put them in the GitHub repository by [adding new pages with the GitHub app](https://help.github.com/articles/creating-new-files/). Then I'll change the links to point to the new pages.
